README.md

```diff
@@ -25,10 +25,8 @@ Heroku-Postgresql databses include *FREE* scheduled backups, we just need to ena
 
 In order to help differentiate git branches a little easier prefix your branch with a relevant keyword.
 
-For example:
-
 | Keyword | Meaning        |
 | ------- | -------------- |
 | feature | new addition   |
 | fix     | fix/adjustment |
-| style   | Style changes  |
```

Comment:
Can we keep `style`?

I would expect it to be used for changes that did not change the functionality of the application, but changed how it looks.